integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -120,14 +123,14 @@ spec:
               name: {{ $val | lower }}-{{ $key | replace "_" "" }}
                         {{- end}}
                       {{- end}}
-                    {{- end}}
                     {{- else}}
             - mountPath: {{ .path }}
               name: {{ .alias | lower }}
                     {{- end}}
-                  {{- end}} 
+                  {{- end}}
                 {{- end}} 
-              {{- end}}
+              {{- end}} 
+            {{- end}} 
```

Comment:
Is there one extra `{{- end}}`? What does this match to?

Reply:
This matches {{- if .mediumtype }}. I think the number of {{- end}} is the same.